lib/tasks/coverage_report.rake

```diff
@@ -7,17 +7,15 @@ namespace :coverage do
   desc "Collates all result sets generated by the different test runners"
   task :report do
     require 'simplecov'
-    require 'simplecov_json_formatter'
-    unless ENV['CI']
-      puts 'This task is intended to be run only on the CI.'
-      return
-    end
-    SimpleCov.collate(Dir["coverage*/.resultset.json"]) do
-      formatter SimpleCov::Formatter::MultiFormatter.new([
-        SimpleCov::Formatter::JSONFormatter,
-        SimpleCov::Formatter::HTMLFormatter
-      ])
-      refuse_coverage_drop
+
+    SimpleCov.coverage_dir('coverage_results')
```

Comment:
To make it easier to identify the collated results, call the new directory `coverage_results`